// // 5. Create a promise that rejects after 1 
// // second with a specific error message.


// const rejectingPromise = new Promise((_, reject)=>{
//     setTimeout(()=>{
//         reject('error message, :9')
//     }, 1000)
// })


// rejectingPromise.catch((error)=>{
//     console.log(error)
// })